modules/ossm-member-roll.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/installing-ossm.adoc
+
+[id="ossm-member-roll_{context}"]
+= Configure the {ProductName} member roll
+
+You must create a `ServiceMeshMemberRoll` resource named `default` associated with the {ProductShortName} in the same project as the `ServiceMeshControlPlane`.
+
+[WARNING]
+====
+If Container Network Interface (CNI) plugin is enabled, manual sidecar injection will work, but pods will not be able to communicate with the control plane unless those pods are specified in the `ServiceMeshMemberRoll` resource.
+====
+
+[NOTE]
+====
+The member projects are only updated if the {ProductShortName} control plane installation succeeds.
+====
+
+* You can add any number of projects, but a project can only belong to *one* `ServiceMeshMemberRoll`.
+
+The `ServiceMeshMemberRoll` is deleted when its corresponding `ServiceMeshControlPlane` resource is deleted.
+
+[id="ossm-member-roll-console_{context}"]
+== Configure the member roll from the {product-title} web console
+
+Follow this procedure to add the Bookinfo project to the {ProductShortName} member roll by using the web console.
+
+.Prerequisites
+* An installed, verified {ProductName} Operator.
+
+.Procedure
+
+. Log into the {product-title} web console.
+
+. Click to *Home* -> *Projects*.
+
+. Click *Create Project*.
+
+. Enter a *Project Name* (for example, *bookinfo*), a *Display Name*, and a *Description*, then click *Create*.
+
+. Click *Catalog* -> *Installed Operators*.
+
+. Click the *Project* menu and choose `openshift-operators` from the list.
+
+. Click the *Istio Service Mesh Member Roll* link under *Provided APIs* for the *{ProductName}* Operator.
+
+. Click *Create Service Mesh Member Roll*.
```

Comment:
@geekspertise 
Click on "All Instances", then "Create New" and select *Create Service Mesh Member Roll*.

Reply:
@geekspertise @rcernich @brian-avery 
Actually at this point, if the OLM is used to install all the operators, there is no existing Service Mesh Member Roll and I get "404: Page Not Found".  The only way to add the member roll is via "Add->Import YAML" or use the CLI altogether.

Reply:
The SMCP would have to exist before we can edit the member roll.  The order of operations in our install doc needs to be updated so that the member roll steps for OLM and CLI are after the SMCP creation.

Reply:
Note the "404" @travisrogers05 has been seeing seems to be an issue with OLM and how it configures the namespaces, it may be necessary to refresh the console before you are presented with the *Create Istio Service Mesh Member Roll* button.

Reply:
@travisrogers05 and @knrc I added the following note:

[NOTE]
====
It can take a short time for the Operator to finish creating the projects, therefore you may need to refresh the screen before the web console presents the *Create Istio Service Mesh Member Roll* button.
====